zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/TagsValidator.java

```diff
@@ -11,26 +11,28 @@
 import static io.camunda.zeebe.gateway.rest.validator.ErrorMessages.ERROR_MESSAGE_INVALID_TAG_FORMAT;
 
 import io.camunda.zeebe.util.TagUtil;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Set;
 
 public class TagsValidator {
 
-  public static void validate(final Set<String> tags, final List<String> violations) {
+  public static List<String> validate(final Set<String> tags) {
```

Comment:
🔧 the signature doesn't really indicate what the returned object represents. Not a big deal, but maybe something like `extractValidationErrors` might make more sense